Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -59,6 +59,7 @@
 from L1Trigger.Configuration.L1Trigger_EventContent_cff import *
 from RecoVertex.BeamSpotProducer.BeamSpot_EventContent_cff import *
 from CommonTools.ParticleFlow.EITopPAG_EventContent_cff import EITopPAGEventContent
+from RecoCTPPS.TotemRPLocal.totemRPLocalReconstruction_cff import *
```

Comment:
@slava77 Both of your comments are addressed in #14616. RecoCTPPS.TotemRPLocal.totemRPLocalReconstruction_cff has been removed from this PR.